src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/task.py

```diff
@@ -106,7 +107,13 @@ def acr_task_create(cmd,  # pylint: disable=too-many-locals
         source_control_type = SourceControlType.github.value
 
     source_triggers = None
-    if commit_trigger_enabled:
+    source_trigger_events_ = []
```

Comment:
nit: remove trailing _ `source_trigger_events`

Reply:
Changed in most recent update.